inji-verify-sdk/README.md

```
@@ -2,6 +2,27 @@
+3. **Camera permissions** - For QR scanning features
+
+### Backend Requirements:
+
+Your backend must support the OpenID4VP protocol. You can either:
+
+- Use the official `inji-verify-service`
+- Build your own following [this specification](https://openid.net/specs/openid-4-verifiable-presentations-1_0-ID3.html)
+
+**Important:** Your backend URL should look like:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)